java/test/jmri/jmrit/display/layoutEditor/LayoutEditorToolsTest.java

```diff
@@ -597,10 +685,14 @@ public void setUp() throws Exception {
         if (!GraphicsEnvironment.isHeadless()) {
             JUnitUtil.resetProfileManager();
 
+            // set default string matching comparator to one that exactly matches and is case sensitive
+            Operator.setDefaultStringComparator(new Operator.DefaultStringComparator(true, true));
```

Comment:
It is entirely possible that this operation on the operator has global implications. This actually MIGHT be causing failures in tests outside this test class.

Make sure you reset this to the previous default when cleanup occurs.